packages/data-context/test/unit/data/ProjectConfigIpc.spec.ts

```diff
@@ -38,10 +38,9 @@ describe('ProjectConfigIpc', () => {
   context('forkChildProcess', () => {
     // some of these node versions may not exist, but we want to verify
     // the experimental flags are correctly disabled for future versions
-    const NODE_VERSIONS = ['18.20.4', '20.17.0', '20.19.0', '22.0.0', '22.7.0', '22.11.4', '22.12.0', '22.15.0']
-    const experimentalDetectModuleIntroduced = '22.7.0'
-    const experimentalRequireModuleIntroduced = '22.12.0'
-    const minorPatchExperimentalModuleIntroduced = '>= 20.19.0 < 21.0.0'
+    const NODE_VERSIONS = ['18.20.4', '20.17.0', '20.5.1', '20.6.0', '22.14.0']
```

Comment:
make sure these are the same on the right. remove node 18
```suggestion
    const NODE_VERSIONS = ['18.20.4', '20.17.0', '20.5.1', '20.6.0', '22.14.0']
```

Reply:
addressed in 0a526aab83